### PR TITLE
Fixed issues with typegen generation for quotes in actions, delays, guards, services and event types

### PR DESCRIPTION
**Pull request by Andarist**
fixes https://github.com/statelyai/xstate-tools/issues/127

### CODE REVIEW

**Review by mattpocock — APPROVED** [review]
Nice

File: apps/cli/src/__tests__/__examples__/action-single-quote.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { createMachine } from "xstate";
+
+createMachine({
+  tsTypes: {} as import("./action-single-quote.typegen").Typegen0,
+  schema: {
+    events: {} as { type: "PING" },
+  },
+  on: {
+    PING: {
+      actions: "say 'We surely shall see the sun shine soon'",
```
Comment (erikras) [inline apps/cli/src/__tests__/__examples__/action-single-quote.ts:10]: WTF sort of British aristocrat are you, Andarist?

File: apps/cli/src/__tests__/__examples__/guard-single-quote.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { createMachine } from "xstate";
+
+createMachine({
+  tsTypes: {} as import("./guard-single-quote.typegen").Typegen0,
+  schema: {
+    events: {} as { type: "PING" },
+  },
+  on: {
+    PING: {
+      cond: "it's play time",
```
Comment (erikras) [inline apps/cli/src/__tests__/__examples__/guard-single-quote.ts:10]: YES!! 💪

File: apps/cli/src/__tests__/__examples__/action-single-quote.ts (hunk 1 above)
Comment (Andarist) [reply apps/cli/src/__tests__/__examples__/action-single-quote.ts:10]: one that googles "tongue twister" 🤣

**Pull request merged**